build/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.9
+
+RUN apk update && \
+    apk add gcc openjdk8 python3 python3-dev make musl-dev libffi-dev openssl openssl-dev
+
+WORKDIR /src
+COPY requirements.txt /src
+
+RUN pip3 install -r requirements.txt
```

Comment:
Why do this when we have the `pip3 Install -e .` later?

Reply:
This is useful for development purposes - mainly speed. Iterating on this codebase without modifying `requirements.txt` file would result in invalidating docker layer that container installation. This keeps `pip` dependencies installed.

Reply:
Thank you - makes sense - please could you add a comment which says that so that someone like me doesn't come along and change this later.